digid_eherkenning/oidc/migrations/0004_migrate_config_to_claimfield.py

```diff
@@ -0,0 +1,258 @@
+# Generated by Django 4.2.13 on 2024-06-11 13:43
+
+from typing import Any, Callable
+
+from django.conf import settings
+from django.core.cache import caches
+from django.db import migrations, models, transaction
+
+import mozilla_django_oidc_db.fields
+
+
+def flush_cache():
+    cache_name = getattr(settings, "SOLO_CACHE", None)
+    if not cache_name:
+        return
+    caches[cache_name].clear()
+
+
+def operation_factory(model: str, mappings: dict[str, str]) -> migrations.RunPython:
+
+    def _action_factory(transformer: Callable[[Any], None]):
+        def _run_python_action(apps, _) -> None:
+            ConfigModel = apps.get_model("digid_eherkenning_oidc_generics", model)
```

Comment:
Is this app name correct? I know the label of `OIDCAppConfig` is `digid_eherkenning_oidc_generics`, but the name is `digid_eherkenning.oidc` :thinking:

Reply:
yeah, I deliberately kept it the same in the app config so that we can sort out the DB tables/migrations. For Open Forms, this will be easy, for Open Inwoner, it will take some more effort but I'll handle that in due time.

I'm planning to add some more migrations afterwards to rename the tables and have a better app label (`digid_eherkenning_oidc`)